content/docs/300-on-brand-menu-api/300-api-keys.mdx

```diff
@@ -23,6 +23,8 @@ API keys can be created and managed from the Developer Portal’s [API keys](htt
 
    ie. `a98b440c16414f9c8a8c3fc239ce84fe`
 
+<Callout>**Single API Key vs Many API Keys**: Although it's possible to create a unique API Key for each of your locations and menus, creating just a single API Key for all locations and menus simplifies API Key management and takes the guess-work out of remembering which key goes with which menus and locations.</Callout>
```

Comment:
```suggestion
<Callout>**Single vs multiple API Keys**: You can create a single API key for multiple locations and menus to avoid remembering which key goes with which menus and locations.</Callout>
```